src/app/shared/components/button/button.component.spec.ts

```diff
@@ -18,4 +18,17 @@ describe('ButtonComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should emit output event', () => {
+    let emittedValue: void;
+
+    component.clicked.subscribe((value: void) => {
+      emittedValue = value;
```

Comment:
If the value is void you can test by creating additional variable for the test
Like this
Create variable
`let emitted = false`
Then change it
`component.clicked.subscribe(() => {
    emitted = true;
  });`
And check
`expect(emitted).toBe(true);`

Reply:
thanks, done